server/controllers/controller.js

```diff
@@ -68,7 +77,7 @@ class Cloths {
         if (result.error){
             return res.status(400).send({
                 status: 400,
-                message: 'only positive numbers are allowed in the Cloth Id field'
+                message: 'only positive numbers are allowed in the Cloth Id fielld'
```

Comment:
The work is good, only this typo on this line needs to be changed

Reply:
Thank You Joe , I have changed it